js/StickyScrollBarComponent.js

```diff
@@ -0,0 +1,138 @@
+class StickyScrollBarComponent {
+
+    /**
+     * StickyScrollBarComponent
+     * @constructor
+     * @param {jQuery} $rootWindow - jQuery wrapper of the window object
+     * @param {jQuery} $html - jQuery wrapper of the html node
+     */
+    constructor ($rootWindow, $html) {
+        this.$elementWithStickyScrollBar = {};
```

Comment:
Why do we initialise this value with an object literal?

Reply:
Good question... removing